### PR TITLE
Gift week email alert api

### PR DESCRIPTION
**Pull request by peteglondon**
* Add a wrapper for the `email-alert-api` `GET /subscriber-lists/metrics/path/to/content` endpoint
* Add a test helper to the above

### CODE REVIEW

File: test/test_helpers/email_alert_api_test.rb
Comment (leenagupte) [inline test/test_helpers/email_alert_api_test.rb:1]: 😍

Comment (peteglondon) [reply test/test_helpers/email_alert_api_test.rb:1]: Amended in https://github.com/alphagov/gds-api-adapters/pull/1227/commits/ac80426190ef5146380ae663e2e01a36e30fbb69

File: test/email_alert_api_test.rb [hunk 1]
```diff
@@ -549,6 +549,28 @@
...
+      api_response = api_client.get_subscriber_list_metrics(path: "/the/path")
+      assert_equal(200, api_response.code)
+      parsed_body = api_response.to_h
+      assert_equal({ "subscriber_list_count" => 5, "all_notify_count" => 12 }, parsed_body)
+    end
+
+    it "raises 404 when not found" do
+      stub_get_subscriber_list_metrics_not_found(path: "/the/path")
+
+      assert_raises GdsApi::HTTPNotFound do
```
Comment (leenagupte) [inline test/email_alert_api_test.rb:568]: Should this return `0` for both metrics rather than a 404? 
cc @KludgeKML

**Review by leenagupte — COMMENTED** [review]
This looks good, and I love the tests and test helpers.
I have one question based on the changes we made to the metrics response today.

File: test/email_alert_api_test.rb (hunk 1 above)
Comment (peteglondon) [reply test/email_alert_api_test.rb:568]: I left the test in because that's what the adapter will do in the event of a 404.

**Review by KludgeKML — APPROVED** [review]
LGTM

**Pull request merged**